tests/unit/test_duckdb.py

```diff
@@ -17,17 +17,6 @@ def duckdb():
     os.remove(DATABASE_PATH)
 
 
-def test__check_if_schema_exists(duckdb):
-
-    duckdb.run(f"DROP SCHEMA IF EXISTS {SCHEMA}")
-    assert not duckdb._check_if_schema_exists(SCHEMA)
-
-    duckdb.run(f"CREATE SCHEMA {SCHEMA}")
-    assert not duckdb._check_if_schema_exists(SCHEMA)
-
-    duckdb.run(f"DROP SCHEMA {SCHEMA}")
-
-
```

Comment:
How do you know the test doesn't work and it's not a problem with `_check_if_schema_exists()`?

Reply:
We talked about it before - this PR was supposed only to fix tests, there is a separate issue ( #505 ) for fixing `_check_if_schema_exists()`

Reply:
Cool, can you put this info in the PR description?